kubetest/e2e.go

```diff
@@ -202,6 +203,18 @@ func run(deploy deployer, o options) error {
 				return tester.Test()
 			}))
 
+		} else if o.deployment == "conformance" {
```

Comment:
this whole block probably can be a switch statement, `o.nodeTests` can be `o.deployment == "node"`

Reply:
Now that I've written modular deployments, I want to modularize *existing* deployments. If that refactor goes well, we won't even need a switch statement here.

Instead we'll create the appropriate testing object at construction, and this code will be like:

`			if err := control.XmlWrap(&suite, "IsUp", deploy.IsUp); err != nil {
				errs = util.AppendError(errs, err)
			}
			errs = util.AppendError(errs, control.XmlWrap(&suite, "Test", func() error {
				return tester.Test()
			}))`